nemo/collections/tts/fastspeech/loss.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2019 NVIDIA Corporation
```

Comment:
same comments: proper header, and add_port_docs decorator for input/output ports